Cargo.toml

```diff
@@ -177,6 +177,8 @@ axum-extra = { version = "0.9", default-features = false }
 papaya = { version = "0.2", default-features = false }
 tempfile = { version = "3", default-features = false }
 fs4 = { version = "0.13.1", default-features = false }
+fnv = { version = "1.0.7", default-features = false }
```

Comment:
Super small note: we generally prefer to scope this to the minimum version necessary, kind of similar to what Go does by default.

Cargo will assume here, for example, that we _need_ `fnv` to be at least `1.0.7`, even if `1.0.0` would be fine. Even if we only put `1`, Cargo will use the latest version it reasonably can, will adhering to semver.

So for `fnv`, we can say `1`.